tests/test_cli.py

```diff
@@ -19,10 +23,22 @@
 
 import zeekclient as zc
 
+# A context guard to switch the current working directory.
+# With 3.11 this can go and become contextlib.chdir():
+@contextmanager
+def setdir(path):
```

Comment:
This could also be an explicit `cwd=tmpdir` on the cmake/make subprocess invocations to avoid the extra import/code. But no real strong opinion.